### PR TITLE
adjust fd file for issue #879

### PR DESCRIPTION
**Pull request by davidcarlisle**
# Internal housekeeping

## Status of pull request

- Ready to merge

## Checklist of required changes before merge will be approved
- [X] Test file(s) added
- [X] Version and date string updated in changed source files
- [X] Relevant `\changes` entries in source included
- [X] Relevant `changes.txt` updated
- [X] `ltnewsX.tex` (and/or `latexchanges.tex`) updated

### CODE REVIEW

File: base/changes.txt [hunk 1]
```diff
@@ -6,6 +6,10 @@ completeness or accuracy and it contains some references to files that
 are not part of the distribution.
 ================================================================================
 
+2022-07-10  David Carlisle  <David.Carlisle@latex-project.org>
+
+	* cmfonts.fdd: avoid EC ss fonts at small design sizes. Scale 8pt font. gh/879
```
Comment (FrankMittelbach) [inline base/changes.txt:11]: should say why eg  , because they are broken            or something

File: base/doc/ltnews36.tex [hunk 2]
```diff
@@ -0,0 +1,152 @@
+% \iffalse meta-comment
+%
```
Comment (FrankMittelbach) [inline base/doc/ltnews36.tex:2]: I thought there is already a ltnews36 ... might be that it is in the umerged branch made by Ulrike (which should get merged)

File: base/doc/ltnews36.tex [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+{\hyphenpenalty=10000 \exhyphenpenalty=10000 \spaceskip=3.33pt \hbadness=10000
+\tableofcontents}
+
+\setlength\rightskip{0pt plus 3em}
+
+
+
+
+\subsection{EC sans serif at small sizes}
+The EC (T1 encoded Computer Modern) fonts have errors at small sizes.
```
Comment (FrankMittelbach) [inline base/doc/ltnews36.tex:144]: Computer Modern Sans

**Review by FrankMittelbach — APPROVED** [review]
small scale docu suggestions, and please check why you had to add ltnews36 and how that fits in with an existing one (in some branch)

File: base/changes.txt (hunk 1 above)
Comment (davidcarlisle) [reply base/changes.txt:11]: I'll add some words

File: base/doc/ltnews36.tex (hunk 2 above)
Comment (davidcarlisle) [reply base/doc/ltnews36.tex:2]: not in devlop, and I wanted to add a para somewhere. We should be able to handle that when merging the other branch (unless that is ready to go in first, in which case I'll rebase this after that goes in)

Comment (davidcarlisle) [reply base/doc/ltnews36.tex:2]: Ulrike's PR now merged, touched exactly same set of files as this, cleaning up merge conflicts here...

**Pull request merged**